index.js

```diff
@@ -96,7 +96,13 @@ function formatNode(node) {
     if (node.children && node.children.length) {
         log += dim('[') + yellow(node.children.length) + dim(']');
     } else {
-        log += dim(': \'') + green(node.value) + dim('\'');
+        log += dim(': ') + green(JSON.stringify(node.value));
```

Comment:
@muraken720 Why did you opt to use `JSON.stringify` here versus the previous system? I really liked the single quotes, and I don’t think `node.value` can/should be non-string!